src/Augurk.Api/Augurk.Api.csproj

```diff
@@ -1,155 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since ASP.NET Core has its dedicated wwwroot folder for static content, I didn't think it was useful anymore to have a separate Api project that contains the WebAPI stuff, so I've merged the Api project into the main project. I think this will make things easier to find for new developers as well. @marktaling What do you think?